portfolio/src/main/webapp/index.html

```diff
@@ -40,5 +40,24 @@ <h1>Mauricio's Portfolio</h1>
       <button onclick="randomPic()">One of my favorite photos</button>
       <img id="favorite-pic" style="display: none;">
     </div>
+    <form action="/form-handler" method="POST">
+        <p>Enter some text here:</p>
+        <textarea name="text-input">hello world</textarea>
+        <br/>
+        <input type="submit" />
+    </form>
+    <footer>
+        <form action="/suggestions" method="POST">
```

Comment:
Where is the suggestions servelet? It is working your website, so I am guessing it is not in this PR?

Reply:
Should be added now